stingray/lightcurve.py

```diff
@@ -12,12 +12,16 @@
 from stingray.utils import simon, assign_value_if_none
 from stingray.gti import cross_two_gtis, join_gtis, gti_border_bins
 from stingray.gti import check_gtis
+from astropy.stats import poisson_conf_interval
 
 __all__ = ["Lightcurve"]
 
+valid_statistics = ["poisson", "gauss", None]
+
 
 class Lightcurve(object):
-    def __init__(self, time, counts, input_counts=True, gti=None):
+    def __init__(self, time, counts, err=None, input_counts=True,
+                 gti=None, statistic=None):
         """
```

Comment:
Actually, eliminating that `None` makes the code less complex, and I like that. If one wants zero errorbars, they will specify `err=np.zeros(whatever)` :)